docs/readthedocs/source/doc/PythonAPI/LLM/langchain.rst

```diff
@@ -1,144 +1,164 @@
 BigDL-LLM LangChain API
 =====================
 
-llm.langchain.embeddings.bigdlllm
+LLM Wrapper of LangChain
 ----------------------------------------
 
-.. autoclass:: bigdl.llm.langchain.embeddings.bigdlllm.LlamaEmbeddings
-    :members:
-    :undoc-members:
-    :show-inheritance:
-    :exclude-members: ggml_model , ggml_module
+Hugging Face ``transformers`` Format
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    .. automethod:: validate_environment
-    .. automethod:: embed_documents
-    .. automethod:: embed_query 
+BigDL-LLM provides ``TransformersLLM`` and ``TransformersPipelineLLM``, which implement the standard interface of LLM wrapper of LangChain.
 
+.. tabs::
 
-.. autoclass:: bigdl.llm.langchain.embeddings.bigdlllm.BloomEmbeddings
-    :members:
-    :undoc-members:
-    :show-inheritance:
-    :exclude-members: ggml_model , ggml_module
+    .. tab:: AutoModel
 
-    .. automethod:: validate_environment
-    .. automethod:: embed_documents
-    .. automethod:: embed_query
+        .. automodule:: bigdl.llm.langchain.llms.transformersllm
```

Comment:
The `return` syntax seems to be wrong for `from_model_id` and `from_model_id_low_bit`

Reply:
> The `return` syntax seems to be wrong for `from_model_id` and `from_model_id_low_bit`

yeah, have fixed.